src/services/account/usePlanData.ts

```diff
@@ -12,6 +12,7 @@ export const TrialStatuses = {
 
 const PlanDataSchema = z
   .object({
+    hasPrivateRepos: z.boolean().nullish(),
```

Comment:
looking at https://github.com/codecov/codecov-api/blob/main/graphql_api/types/owner/owner.graphql#L18
it's not nullish

Reply:
Good catch! Gonna adjust this

Reply:
Since this is a required field now, had to adjust other tests to add it, hence why the files changed blew up a bit, just letting you know 👌